src/js/clickable-component.js

```diff
@@ -155,6 +167,7 @@ class ClickableComponent extends Component {
       }
       this.on(['tap', 'click'], this.handleClick);
       this.on('keydown', this.handleKeyDown);
+      this.on('languagechange', this.handleLanguageChange);
```

Comment:
I don't think that we need a check for the method because it inherits from Component which has an abstract method for it.
However, why not have this line in Component itself? That way all components are automatically hooked up and just need to implement the `handleLanguagechange` method?